web/e2e/specs/browse/browse-data.ts

```diff
@@ -38,10 +38,10 @@ export default function (qaProjectDir) {
     });
 
     it('should verify navigation to the next, last, previous, first pages', async function () {
-      await browsePage.paginateClick('next');
-      await browsePage.paginateClick('previous');
-      await browsePage.paginateClick('last');
-      await browsePage.paginateClick('first');
+      await browsePage.clickPaginate('next');
+      await browsePage.clickPaginate('previous');
+      await browsePage.clickPaginate('last');
+      await browsePage.clickPaginate('first');
     });
```

Comment:
There is no validation that after clickPaginate we really land at the intended page. So if this test is really validating the pagination button isnt dead then its fine. Otherwise IMO we need some validation.

Reply:
Added validation for the number of docs > 0.  Thanks.